rwlib/source/loaders/LoaderIMG.cpp

```diff
@@ -5,9 +5,9 @@
 LoaderIMG::LoaderIMG() : m_version(GTAIIIVC), m_assetCount(0) {
 }
 
-bool LoaderIMG::load(const std::string& filename) {
-    auto baseName = filename;
-    auto extpos = filename.find(".img");
+bool LoaderIMG::load(const rwfs::path& filepath) {
+    auto baseName = filepath.string();
+    auto extpos = baseName.find(".img");
```

Comment:
Could this be achieved with path::stem() instead?

Reply:
I changed the code with `replace_extension`.
Too bad C++ does not have `with_extension` (like Python has with [with_suffix](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.with_suffix))